docs/content/getting-started/releases.mdx

```diff
@@ -1,28 +1,41 @@
 
-## Breaking Changes & Deprecations
+Our public, stable Python API includes:
 
-We are in the 0.y.z of [SemVer](https://semver.org/). While technically this is "anything goes", we provide guidance to set expectations around deprecations and breaking changes:
+- All classes, functions, and variables that are exported at the top-level of the `dagster` package, unless they're marked experimental (more on this [below](#experimental-apis)).
+- Public, non-[experimental](#experimental-apis) methods and properties of public, stable classes. Public methods and properties are those show up in the [API reference](/\_apidocs). Within the codebase, they're marked with a `@public` decorator.
 
-- Upgrading to a new dot version within a minor release, e.g. 0.8.1 → 0.8.2, should never result in broken code. If a new dot version does break your code, please help us out by filing an issue on [Github](https://github.com/dagster-io/dagster/issues).
-- An exception to the above is “experimental APIs”, which allow us to offer new APIs to users and rapidly iterate based on their feedback. Experimental APIs will emit a warning whenever used, and may be broken even in dot releases.
-- As often as possible, deprecation warnings will precede removals. E.g. if we’re currently on version 0.8.5, and we wish to remove an API, we’ll start issuing a deprecation warning when the API is used, and then we’ll omit it from 0.9.0.
+### Experimental APIs
+
+The "experimental" marker allows us to offer new APIs to users and rapidly iterate based on their feedback. Experimental APIs are marked as such in the [API reference](/\_apidocs) and usually raise an <PyObject object="ExperimentalWarning" /> when used.
+
+Experimental APIs may change or disappear within any release, but we try to give warning for experimental APIs that have been around for a long time.
+
+### Deprecated APIs
+
+The "deprecated" marker indicates that we recommend avoiding an API, usually because there's a preferred option that should be used instead.
+
+Like non-deprecated public stable APIs, deprecated public stable APIs will not break within any major release after 1.0.
+
+## Dagster integration libraries
+
+Dagster's integration libraries are still in the 0.y.z of [SemVer](https://semver.org/). While technically this is "anything goes", we are conservative about making changes and provide guidance about when to expect breaking changes:
+
+- Upgrading to a new dot version within a minor release, e.g. 0.8.1 → 0.8.2, should never result in broken code.
+- An exception to the above is "experimental" APIs, as [described above](#experimental-apis).
+- As often as possible, deprecation warnings will precede removals. E.g. if we’re currently on version 0.8.5, and we wish to remove an API, we’ll start issuing a deprecation warning when the API is used, and then we'll omit it from 0.9.0.
```

Comment:
> when the API is used

Meaning the API will return a deprecation warning?

Reply:
Python has a "warnings" feature: https://docs.python.org/3/library/warnings.html. I'll link it from here and try to disambiguate from other uses of warning in this doc.

Reply:
Gotcha. I think this guy could be improved for clarity, so I wanted to know what the user should expect to happen so it could be included:

> E.g. if we’re currently on version 0.8.5, and we wish to remove an API, we’ll start issuing a deprecation warning when the API is used, and then we'll omit it from 0.9.0.

Maybe something like:

> E.g. If the current version is 0.8.5 and we want to remove an API, we'll issue a deprecation warning when the API is used and remove it from 0.9.0.